Kucoin.Net/Converters/StopConditionConverter.cs

```diff
@@ -16,5 +18,13 @@ internal class StopConditionConverter : BaseConverter<StopCondition>
             new KeyValuePair<StopCondition, string>(StopCondition.Loss, "loss"),
             new KeyValuePair<StopCondition, string>(StopCondition.Loss, "down"),
         };
+
+        public override object? ReadJson(JsonReader reader, Type objectType, object? existingValue, JsonSerializer serializer)
+        {
+            object? valueObject = base.ReadJson(reader, objectType, existingValue, serializer);
+            if (valueObject == null)
```

Comment:
Overrode ReadJson to return otherwise useless StopCondition.None rather than Null
Since ReadJson always returns a value from the enum, StopCondition variables don't need to be nullable.